source/virtualBuffers/MSHTML.py

```diff
@@ -53,6 +53,18 @@ def _normalizeControlField(self,attrs):
 		accRole=attrs.get('IAccessible::role',0)
 		accRole=int(accRole) if isinstance(accRole,basestring) and accRole.isdigit() else accRole
 		nodeName=attrs.get('IHTMLDOMNode::nodeName',"")
+		
+# 		idAttribute=attrs.get('HTMLAttrib::id')
+# 		nameAttribute=attrs.get('HTMLAttrib::name')
+# 		if (idAttribute and nameAttribute):
+# 			customLabel=self.obj.rootNVDAObject.getCustomLabel(idAttribute+nameAttribute)
+# 		elif (not idAttribute):
+# 			customLabel=self.obj.rootNVDAObject.getCustomLabel(nameAttribute)
+# 		elif (not nameAttribute):
+# 			customLabel=self.obj.rootNVDAObject.getCustomLabel(idAttribute)
+# 		if customLabel:
+# 			attrs["name"]=customLabel

```

Comment:
Is there a reason why the name attribute couldn't have been set in the MSHTML backend c++ code, probably when setting contentString? Or is this different.